apps/token-manager/app/src/screens/Details.js

```diff
@@ -0,0 +1,193 @@
+          <VestingInfoBoxes
+            selectedHolder={selectedHolder}
+            tokenSymbol={tokenSymbol}
+            vestings={vestings}
+          />
+        }
+        invert="horizontal"
+      />
+    </React.Fragment>
+  )
+}
+
+function VestingContent({ vesting, tokenSymbol }) {
+  const { layoutName } = useLayout()
+  const compact = layoutName === 'small'
+  const vestingInfo = useVestedTokensInfo(vesting)
+
+  return (
+    <div
+      css={`
+        display: flex;
+        padding: ${3 * GU}px 0;
+        width: 100%;
+        flex-direction: ${compact ? 'column' : 'row'};
+      `}
+    >
+      <div
+        css={`
+          flex-shrink: 0;
+          flex-grow: 0;
+          width: ${33 * GU}px;
+          display: flex;
+          align-items: center;
+        `}
+      >
+        {formatTokenAmount(vesting.amount, 18, { symbol: tokenSymbol })}
+      </div>
+      <div
+        css={`
+          flex-grow: 1;
+        `}
+      >
+        <SummaryBar
+          vestingInfo={vestingInfo}
+          css={`
+            margin-top: 0;
+            margin-bottom: ${2 * GU}px;
+          `}
+        />
+        <SummaryRows vestingInfo={vestingInfo} tokenSymbol={tokenSymbol} />
+      </div>
+    </div>
+  )
+}
+
+function ExpandableContent({ vesting, tokenSymbol }) {
+  const theme = useTheme()
+  const vestingInfo = useVestedTokensInfo(vesting)
+  return (
+    <div
+      css={`
+        display: flex;
+        padding: ${3 * GU}px ${4 * GU}px;
+        width: 100%;
+        justify-content: space-between;
+      `}
+    >
+      <div>
+        <ContentBox>
+          <label
+            css={`
+              color: ${theme.surfaceContentSecondary};
+            `}
+          >
+            START DAY
+          </label>
+          <p>{formatDate(toISODate(vesting.start))}</p>
+        </ContentBox>
+        <ContentBox>
+          <label
+            css={`
+              color: ${theme.surfaceContentSecondary};
+            `}
+          >
+            END DAY
+          </label>
+          <p>{formatDate(toISODate(vesting.vesting))}</p>
+        </ContentBox>
+      </div>
+      <div>
+        <ContentBox>
+          <label>VESTING PERIOD</label>
+          <p>
+            {formatDistanceStrict(
+              toISODate(vesting.vesting),
+              toISODate(vesting.start)
+            )}
+          </p>
+        </ContentBox>
+        <ContentBox>
+          <label>VESTING CLIFF</label>
+          <p>
+            {formatDistanceStrict(
+              toISODate(vesting.cliff),
+              toISODate(vesting.start)
+            )}
+          </p>
+        </ContentBox>
+      </div>
+      <div>
+        <ContentBox>
+          <label>AVAILABLE TO TRANSFER</label>
+          <p>
+            {formatTokenAmount(vestingInfo.unlockedTokens, 18, {
+              symbol: tokenSymbol,
+            })}
+          </p>
+        </ContentBox>
+      </div>
+    </div>
+  )
+}
+
+function ContentBox({ props, children }) {
```

Comment:
Let's inline this in the other component where it's used.

It's especially awkward because this component styles children elements _implicitly_ via its css selectors.